src/librustc_typeck/check/method/mod.rs

```diff
@@ -195,8 +195,8 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
             ProbeScope::TraitsInScope
         )?;
 
-        if let Some(import_id) = pick.import_id {
-            let import_def_id = self.tcx.hir().local_def_id_from_hir_id(import_id);
+        for import_id in &pick.import_ids {
```

Comment:
```suggestion
        for import_id in pick.import_ids {
```

Reply:
Won't work because of pick.clone() further down :-(
My Rust powers are lacking.